native/packages/react-native-bpk-component-button/stories.js

```diff
@@ -0,0 +1,257 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  ScrollView,
+  Image,
+  Platform,
+} from 'react-native';
+
+import PropTypes from 'prop-types';
+
+import { storiesOf } from '@storybook/react-native';
+import { action } from '@storybook/addon-actions';
+
+import BpkText from 'react-native-bpk-component-text';
+
+import BpkButton, { BUTTON_TYPES } from './src/BpkButton';
+
+import ArrowImageSrc from './rightarrow_360.png';
+
+const IOS_TOKENS = require('bpk-tokens/tokens/ios/base.react.native.common.js');
+const ANDROID_TOKENS = require('bpk-tokens/tokens/android/base.react.native.common.js');
+
+const tokens = Platform.select({
+  ios: () => IOS_TOKENS,
```

Comment:
Why aren't we requring in here? The primary reason I started using comon JS was so that we could dynimcally require per platform. I see that this has also regressed for `BpkText`

Reply:
Now changed. I changed it in the first place because ESLint didn't like it - will change it for BpkText in a future PR too.